app/models/container.rb

```diff
@@ -4,6 +4,7 @@ class Container < ActiveRecord::Base
 
   has_one    :container_group, :through => :container_definition
   delegate   :ext_management_system, :to => :container_group
+  delegate   :container_replicator, :to => :container_group
```

Comment:
Can you check if all these ones can be re-written with `:through =>`? Thanks /cc @abonas